docs/_docs/versions.md

```diff
@@ -5,114 +5,157 @@ layout: post-toc
 redirect_from: /docs/
 ---
 
-# Manage Versions of Your Zapier Integration
+# How to update your integration on the Zapier Developer Platform
 
-Your Zapier integration should be quite nice when first launched—minimal, perhaps, but with the basics covered. The app authentication will let users connect to your API, and the triggers and actions should get data to and from your API.
+## Overview
```

Comment:
I suggest we remove `## Overview`. It'd be better to keep a consistent format across all the documents. Other documents at the same level, such as [Intro](https://platform.zapier.com/docs/zapier-intro) and [Getting Started](https://platform.zapier.com/docs/start), start an overview block right away without an "Overview" caption.